Sources/MapboxNavigation/CameraController.swift

```diff
@@ -74,8 +74,10 @@ class CameraController: NavigationComponent, NavigationComponentDelegate {
     
     func center(on step: RouteStep, route: Route, legIndex: Int, stepIndex: Int, animated: Bool = true, completion: CompletionHandler? = nil) {
         navigationMapView.navigationCamera.stop()
-        // TODO: Verify that camera is positioned correctly.
+
+        let edgeInsets = navigationMapView.safeArea + UIEdgeInsets(top: 10.0, left: 20.0, bottom: 10.0, right: 20.0)
```

Comment:
Would it be possible to extract this `UIEdgeInsets` hardcoded value into separate constant? I think there are already four usages of similar value in other classes.

Reply:
Thanks for the suggestion! I'll work in this part.